lhotse/recipes/mini_librispeech.py

```diff
@@ -48,17 +53,17 @@ def prepare_mini_librispeech(
 
     for part in dataset_parts:
         # Generate a mapping: utt_id -> (audio_path, audio_info, text)
-        metadata = defaultdict(dict)
-        MetaDataType = namedtuple('MetaType', ['audio_path', 'audio_info', 'text'])
+        metadata = {}
         part_path = corpus_dir / part
         for trans_path in part_path.rglob('*.txt'):
             with open(trans_path) as f:
                 for line in f:
                     idx, text = line.split(maxsplit=1)
                     audio_path = part_path / Path(idx.replace('-', '/')).parent / f'{idx}.flac'
                     if audio_path.is_file():
-                        audio_path = str(audio_path)
-                        metadata[idx] = MetaDataType(audio_path=audio_path, audio_info=torchaudio.info(audio_path), text=text)
+                        metadata[idx] = LibriSpeechMetaData(audio_path=audio_path,
+                                                            audio_info=torchaudio.info(str(audio_path))[0],
```

Comment:
I don't know `torchaudio.info` very well; what's in index 0, and then in later indices?